test/scripts/git_clone.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# To run this script from your workspace execute this command
+#   ./test/scripts/run.sh /mnt/blob_mnt /mnt/blobfusetmp ./config.yaml ./v1.cfg 2&> results.txt
```

Comment:
we need to correct these steps here as this might not require these many inputs for git clone.

Reply:
it does need the same inputs